src/cacheServerClient/cacheServerClient.ts

```diff
@@ -76,6 +77,11 @@ export class CacheServerClient implements ICacheServerClient {
     });
   }
 
+  async login(claim: string) {
+    const { data } = await this.httpClient.post<{ token: string }>("/login", { claim });
+    this.httpClient.defaults.headers.common["Authorization"] = `Bearer ${data.token}`;
```

Comment:
Another aspect I thought of is whether or not this is the way that the jwt should be stored. I think this is safest spot as it in browser memory (safer than localStorage or cookie, but has downside that don't persist on new tab or refresh). Is this your understanding as well @dwojno @manihagh ?

Reply:
@jrhender yes, I agree we should add option to store jwt in the cookie as configured option, but still we need to have an option to return token for nodejs apps to be able to login

Reply:
@dwojno Ah, sorry, I wasn't trying to say that we necessarily need an option to store jwt as a cookie. I think the way that it is currently being done in this PR is better than cookie or localStorage for our requirements. I just wanted to confirm that this was your and Mani's view as well.